application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -56,4 +59,26 @@ public static String escapeMarkdown(String text) {
         return afterEscape.replace("\\```", "\\`\\`\\`");
     }
 
+    /**
+     * Converts a slash command text to a mentioned slash command, which you can directly click on
+     * in Discord.
+     *
+     * @param guild the {@link Guild} that contains the command
+     * @param commandPath the command's name with its optional subcommand
+     * @return Formatted string for the mentioned slash command
+     * @throws IllegalArgumentException when the command isn't found in the guild
+     */
+    public static RestAction<String> mentionSlashCommand(Guild guild, String commandPath) {
+        String commandName = commandPath.split(" ", 2)[0];
```

Comment:
y 2?

Reply:
> so it basically does nothing

what u want to split in a one word string anyways

Reply:
if theres nothing to split it'll return an array with the string as first element and thats what we are expecting here so change it back to 1
```suggestion
        String commandName = commandPath.split(" ", 1)[0];
```

Reply:
But then `c.getName().equalsIgnoreCase(commandName)` doesn't work anymore (line 75) if the command path has more than one word. That's why we need to extract the first word in all cases.

Reply:
ohh yea, i'll just just remove the limit at this point